### DISCUSSION TITLE
1.17 / Optifine

### DISCUSSION THREAD

**Discussion by oxodao** [SKCraft/Launcher#442] [I need help!]
Hey all!

I'm trying to add Optifine for a private modpack with my friends. I understand that the Optifine license prevents doing modpacks but as this is targeted at a really small group of people this should not be an issue.

I have a small 1.17.1 modpack (< 25 mods) and I'd like to add Optifine for my low-powered friend's pc, and for shaders for myself with a beefier computer.

Unfortunately it looks like Optifine 1.17 is doing some weird things and export both the modules from Forge. As of 1.17 or something like that, you need a quite recent version of Java which definitely don't allow this.

Do anyone knows how to make them work together nicely ?

Thanks!

**Comment by saitohirga** [comment]
As far as I know, Optifine doesn't support forge right now.

**Reply by oxodao** [reply]
It does, not in the stable version though. Using the prerelease one solved my issue. I've tried with OptiForge but that did not work
